apps/editing-toolkit/editing-toolkit-plugin/wpcom-universal-themes/index.php

```diff
@@ -26,8 +26,22 @@ function is_core_fse_active() {
 		return false;
 	}
 
-	// Now we just check for our own option.
-	return (bool) get_option( ACTIVATE_FSE_OPTION_NAME );
+	// We identify universal themes by assuming that they will all use blockbase as the
+	// default template.
+	$universal_theme_template = defined( 'IS_ATOMIC' ) && IS_ATOMIC
+		? 'blockbase'
+		: 'pub/blockbase';
+
+	// For universal themes, we check for our own option.
+	if ( get_option( 'template' ) === $universal_theme_template ) {
```

Comment:
Don't check the option directly, here. It's better to use the `get_template()` function, will will include any filters set for the template value.

Reply:
Nice catch 👍

Done.